src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -63,3 +73,32 @@ test('testing button text updates with default ad params', () => {
     expect(button.innerText).toBe('Find Nearest Location');
 });
 
+test('Map constructor in Maps API is accessed on button click', () => {
```

Comment:
Perhaps - "map appears on button clicked." The word accessed seems odd in this context

Reply:
The way that I structured this test made it so that I'm not actually checking to see if the map appears in the div element but instead I'm checking if the Map constructor in the Maps API is being accessed/called. Do you think that "Map constructor is called on click" would work better?

Reply:
what about: instance of map is instantiated on button click

Reply:
Ohh I like that! Thanks